src/ethereum/ethNetwork.js

```diff
@@ -317,90 +332,31 @@ export class EthereumNetwork {
 
     let blockHeight = tokenTransfer.attributes.globalRank[0]
     if (blockHeight < 0) blockHeight = 0
+
+    let parentNetworkFee
```

Comment:
Declaring as undefined is technically correct but this is a undesirable change considering it was empty string previously. To avoid other things breaking we should keep as empty string or 0 as the other token-supporting plugins do.